PhoneBook.Vocalnight/PhoneBook.Vocalnight/PhoneBook.Vocalnight.csproj

```diff
@@ -0,0 +1,10 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
```

Comment:
@VocalNight The project won't build because the csproj doesn't have the nuget packages. Could you please update it?